recipe/meta.yaml

```diff
@@ -1,28 +1,31 @@
-{% set version = "0.1.6" %}
+{% set version = "0.2.3" %}
 
 package:
   name: clickhouse-sqlalchemy
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/c/clickhouse-sqlalchemy/clickhouse-sqlalchemy-{{ version }}.tar.gz
-  sha256: 402d112e2c7e8c7f28fd2691c5efaf701c1c19831c10a38c9355a07dd8d308a9
+  sha256: 6df4c125db428b856f8a7ec392190d9a4a2bb281eecb4fca2202e9d49ee4d55f
 
 build:
   number: 0
-  noarch: python
-  script: pip install . --no-deps --ignore-installed --no-cache-dir -vvv
+  # clickhouse-driver isn't available on s390x and aarch64.
+  skip: True  # [py<37 or (linux and (s390x or aarch64))]
+  script: pip install . --no-deps --no-build-isolation -vv
 
 requirements:
   host:
     - pip
+    - python
+    - setuptools
+    - wheel
 
   run:
     - python
-    - six
-    - sqlalchemy
+    - clickhouse-driver >=0.1.2
     - requests
-    - clickhouse-driver >=0.0.14
+    - sqlalchemy >=1.4,<1.5
 
 test:
```

Comment:
The upstream contains functional tests. Do we want to add them?

Reply:
They are not available on the source dist https://pypi.org/project/clickhouse-sqlalchemy/0.2.3/#files
<img width="312" alt="Screenshot 2023-09-25 at 12 21 18" src="https://github.com/AnacondaRecipes/clickhouse-sqlalchemy-feedstock/assets/79282962/b31557c6-52ba-4aef-8212-6fbe9677e8a2">

Reply:
ah, that's what I thought. Thanks